bin/appliance_console

```diff
@@ -104,6 +104,7 @@ module ApplianceConsole
       dbhost      = ManageIQ::ApplianceConsole::DatabaseConfiguration.database_host
       database    = ManageIQ::ApplianceConsole::DatabaseConfiguration.database_name
       evm_running = LinuxAdmin::Service.new("evmserverd").running?
+      messaging   = ManageIQ::ApplianceConsole::MessageServerConfiguration.configured?
```

Comment:
Is this used or was this supposed to be used on line 124 below?

Reply:
I've updated var names for clarity. `messaging` -> `messaging_broker`